### DISCUSSION TITLE
Support for Animated File Formats for Artwork

### DISCUSSION THREAD

**Discussion by sallymanda** [lutris/lutris#5111] [Ideas]
Hey everyone :-) I would love to see support for file formats with animation, specifically I was thinking GIF, APNG, WEBP, and the newer AVIF. The thought occurred to me when browsing SteamGridDB, some of the animated artwork is very cool. Here's a few examples of the type of artwork I had in mind:

[Cyberpunk 2077](https://cdn2.steamgriddb.com/file/sgdb-cdn/grid/58a97c95c23b3d8e1567d444c1e024a0.png)
[Baldur's Gate 3](https://cdn2.steamgriddb.com/file/sgdb-cdn/grid/2ddd2ff467042c2d0d238ddfbc77c0fb.webp)
[Diablo 3](https://cdn2.steamgriddb.com/file/sgdb-cdn/grid/bc0d67a22e5a847af0c9b9349d004880.webp)

**Comment by strycore** [comment]
Would GTK support any of those formats?
